levoitManifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "packageName": "Levoit Air Purifiers",
+  "packageName": "Levoit Drivers",
```

Comment:
I'm worried about changing the package name -- how does that affect the Hubitat Package Manager's ability to update installed packages.